docs/tutorials/ssp_profile_catalog_authoring/ssp_profile_catalog_authoring.md

```diff
@@ -127,32 +163,54 @@ Access control policy and procedures address the controls in the AC family that
 
 In a cyclic operation of `generate-edit-assemble` you would simply be re-writing from and to the same json profile, in which case the `--name` is optional and you can just use `trestle author profile-assemble --markdown profile_md --output my_profile`.  This will assemble the markdown profile contents in directory `profile_md` into a json profile named `my_profile` but it will first use the existing `my_profile` json file as the parent profile and incorporate changes (due to user edits) in the markdown version of the profile.
 
-It's important to note that these operations only apply to the `Adds` in the profile itself - and nothing upstream of the profile is affected.  Nor is anything else in the original profile lost or altered.  In the example above, the section, `## Control my_guidance` was added by editing the generated control - and after `profile-assemble` it ended up as new guidance in the assembled profile.
+It's important to note that these operations only apply to the `Adds` in the profile itself - and nothing upstream of the profile is affected.  Nor is anything else in the original profile lost or altered.  In the example above, the section, `## Control Implementation Guidance` was added by editing the generated control - and after `profile-assemble` it ended up as new guidance in the assembled profile.
+
+As in the other commands, `profile-generate` allows specification of a yaml header, and support of the `--overwrite-header-values` flag.   Also, during assembly with `profile-assemble` the `--set-parameters` flag will set parameter values in the profile for the control based on the header in the control markdown file.
+
+## Use of Sections in `profile-generate` and `profile-assemble`
+
+The addition of guidance sections in the `profile` tools requires special handling because the corresponding parts have both a name and a title, where the name is a short form used as an id in the json schema, while the title is the readable form intended for final presentation.  An example is `ImplGuidance` vs. `Implementation Guidance`.  The trestle authoring tools strive to make the markdown as readable as possible, therefore the headings for sections use the title - which means somehow there is a need for a mapping from the short name to the long title for each section.  This mapping is provide in several ways:  During `profile-generate` you may provide a `--sections "ImplGuidance:Implementation Guidance,ExpEvidence:Expected Evidence"` option that would provide title values for `ImplGuidance` and `ExpEvidence`.  This dictionary mapping is then inserted into the yaml header of each control's markdown.  You may also add this mapping directly to a yaml file that is passed in during `profile-generate`, which is preferable if the list of sections is long.  The sections should be entered in the yaml header in a section titled, `x-trestle-sections`.
```

Comment:
change - "provide in several ways..." to "provided in several ways..."
Last sentence, rathe than. - "The sections should be.." I believe it should be - "The sections would be..."

Reply:
The yaml header is specified with --yaml and I have added text to make that clear.

The --sections is simply a mapping and is intended to include all possible sections that may appear - so I don't think it makes sense to prompt for all of them.  If the user wants certain ones to be prompted for all controls they can use --needed-sections.  That gives them complete control over what they want prompted.  The sections may be found in some controls but not others - and in any where the section is present, the correct title will be displayed based on the mapping.